Add lookup support

derekcaneja opened the pull request:
Add support for checking if a number is a mobile number using twilio lookups api.
Do not send texts to non-mobile numbers.

jperl commented on sms_twilio.js:
```diff
@@ -14,6 +14,59 @@ sms.configure = function (options) {
+
+  var body = 'Welcome to ' + Meteor.settings.APP.NAME + ', ' + user.first_name + '.\n\n' +
+              message + '\n\n' +
+              'Download the app now from http://dispatch.me/get';
```
This is not generic.

jperl replied:
I do not think this method belongs in this package.

jperl commented on sms_twilio.js:
```diff
@@ -14,6 +14,59 @@ sms.configure = function (options) {
+var lookups = new Meteor.Collection('meteor_accounts_sms_lookups');
+
+// Check if a number is a mobile number with twilio lookups.
+var isMobileNumber = function (phone) {
```
:+1: Expose this as `Accounts.sms.lookup`

jperl commented on sms_twilio.js:
```diff
@@ -14,6 +14,59 @@ sms.configure = function (options) {
+  });
+
+  if (response.data) {
+    lookups.insert(_.extend(response.data, { timestamp: new Date() }));
```
Please do a check on `response.data` so we can ensure the schema is correct.

raix replied:
don't think we should add the `+1` logic in this package - would hinder use of non US phone nr.?

jperl replied:
Yeah def should not have the `+1` --- thanks, I missed that.

jperl commented:
I would change this PR to be named "Add lookup support"

derekcaneja committed:
Add lookup support.

Add support for checking if a number is a mobile number using twilio lookups api.
Do not send texts to non-mobile numbers.
